packages/svelte/scripts/process-messages/index.js

```diff
@@ -227,29 +240,31 @@ function transform(name, dest) {
 
 				const value = node.value
 					.split('\n')
-					.map((line) => {
-						if (line === ' * MESSAGE') {
-							return messages[messages.length - 1]
-								.split('\n')
-								.map((line) => ` * ${line}`)
-								.join('\n');
+					.map(
+						/** @param {string} line */ (line) => {
```

Comment:
can we put the assertion on `node.value` instead, so that we don't bork up the formatting of this function? the type of `line` should be inferred. ditto for the `.filter` argument below